js/ecl_expandable.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @file
+ * ECL Expandable behavior for dropdown pattern.
+ */
+(function (ECL, Drupal) {
+  Drupal.behaviors.eclExpandableDropdown = {
+    attach: function attach() {
+      var dropdowns = document.querySelectorAll('[data-ecl-expandable-dropdown]');
```

Comment:
Not sure why we need this?

Reply:
'Expandable' ECL component is doing collapsing only by click on the button. But previously used 'dropdown-legacy' ECL component had collapsing also by the loss of focus. By this js code, I applied this behavior to the 'Dropdown' pattern which now based on 'expandable' ECL component.